client/web/src/features/user/userSaga.ts

```diff
@@ -0,0 +1,20 @@
+import { put, takeLatest, call, all } from 'redux-saga/effects'
+
+import * as actionTypes from './actionTypes'
+import UserService from './userService'
+
+function* fetchUsers() {
+  const users = yield call(UserService.getAll)
```

Comment:
Not consitante with seriesSaga:
`const { data: series } = yield call([serieService, 'getAll'])`

Reply:
User feature is not up to date, I will create a new branch about user is starting